docs/help/faq/index.md

```diff
@@ -1,2 +1,32 @@
 # FAQ
 To answer the most frequently asked questions the following list is constantly updated. Please read through before using the support and communication channels for questions.
+
+## General
+
+**What are tokens?** <br>
```

Comment:
MAybe use headings insteaad of strong , that will help accessibility I think.

Reply:
Sure, we can do that!